File: src/CorporateMessengerWithAgile/Application/Dto/Summary/TaskItemInSprintSummaryDto.cs
using Domain.Entity;

namespace Application.Dto.Summary
{
    public class TaskItemInSprintSummaryDto : BaseDto
    {
        public Guid TaskItemId { get; set; }
        public Guid SprintId { get; set; }
        public TaskItemStatus TaskStatus { get; set; }
    }
}

File: src/CorporateMessengerWithAgile/ReactApp/reactapp.client/src/models/entity/TaskItemInSprintSummaryDto.ts
// src/models/entity/TaskItemInSprintSummaryDto.ts
import { z } from 'zod';
import { BaseDtoSchema } from './BaseDto';
import { GuidSchema } from '../Guid';

export const TaskItemInSprintSummaryDtoSchema = BaseDtoSchema.extend({
    taskItemId: GuidSchema,
    sprintId: GuidSchema,
    taskStatus: z.string().min(1),
    description: z.string(),
});

export type TaskItemInSprintSummaryDto = z.infer<typeof TaskItemInSprintSummaryDtoSchema>;
